README.md

```diff
@@ -9,7 +9,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/sardana-feedsto
 
 Summary: Sardana is a software suite for Supervision, Control and Data Acquisition in scientific installations.
 
-Development: https://github.com/sardana-org/sardana
+Development: https://gitlab.com/sardana-org/sardana
```

Comment:
Could you please check if now it is ok?

Reply:
As there was no other change you didn't really have to revert your commit. Sorry I was probably not super clear.
It's just that the files outside the recipe directory are managed by the bot in general.

Anyway, it's fine.

But to avoid publishing a new package, you should still edit the last commit to have `[ci skip] ***NO_CI***` in the message. I propose that you just squash all the commits and change the message (and force push to your branch).